cs.html

```diff
@@ -451,6 +442,74 @@ <h2>Methods</h2>
                 <em>Return type:</em> <code>Promise&lt;boolean&gt;</code>
               </div>
             </dd>
+            <dt><dfn><code>receiveDatagrams</code></dfn></dt>
+            <dd>
+              <p>Deque and return all QUIC datagrams that have been received.</p>
```

Comment:
If datagrams have been received since the last call to receiveDatagrams(), return a promise resolved with all of the received datagrams.  If not, return a new promise that resolves with the next datagram when the next datagram is received.  

If too many datagrams are queued between calls to receiveDatagrams(), the implementation may drop datagrams and replace them with a null in the sequence of datagrams resolved in the promise.    

receiveDatagrams() may only be called once a time.  If a promised returned from a previous call is still unresolved, the user agent MUST return a new promise rejected with an InvalidStateError.

Reply:
Done